src/store/actions/unlockWallet.js

```diff
@@ -1,15 +1,38 @@
 import { encrypt, decrypt, decryptLegacy } from '../../utils/crypto'
+import { Client } from '@liquality/client'
+import { EthereumRpcProvider } from '@liquality/ethereum-rpc-provider'
 
 export const unlockWallet = async ({ commit, state, dispatch }, { key }) => {
   let wallets = await decrypt(state.encryptedWallets, key, state.keySalt)
 
+  const accKeys = Object.keys(state.accounts);
+
+  let addresses = [];
+
+  for(let i = 0; i < accKeys.length; i++) {
```

Comment:
this can be more readable in a foreach, either `for(const walletId in walletIds)` or `walletIds.forEach(walletId => {})`
same for below loop